lib/utils/Thing.js

```diff
@@ -195,6 +196,22 @@ export class Thing {
 		return this.isComment() && !!this.element.parentElement && this.element.parentElement.classList.contains('nestedlisting');
 	}
 
+	isSimilarToTitle(value: string): boolean {
+		if (!value) {
+			return false;
+		}
+		const title = this.getTitle();
+
+		if (!title) {
+			return false;
+		}
+		const distance = levenshtein.get(value, title);
+		if (value.length > title.length) {
+			return (((value.length - distance) / value.length) >= 0.95);
+		}
+		return (((title.length - distance) / title.length) >= 0.95);
```

Comment:
I think this would be cleaner if you flipped the inequality; i.e. `return distance / title.length < 0.05`

Reply:
Yeah that's cleaner.  I still want it to be inclusive of the case where the string is only 5% different, so in my case it'll be `return distance / title.length <= 0.05;`.  I'll add another commit to change this.